cartography/data/jobs/cleanup/aws_import_ec2_security_groupinfo_cleanup.json

```diff
@@ -1,10 +1,5 @@
 {
   "statements": [
-  {
-      "query": "MATCH (n:EC2SecurityGroup)<-[:RESOURCE]-(:AWSAccount{id: $AWS_ID}) WHERE n.lastupdated <> $UPDATE_TAG WITH n LIMIT $LIMIT_SIZE DETACH DELETE (n)",
```

Comment:
This is covered by autocleanups in models.aws.ec2.keypairs. Additionally the autocleanups also handle a previously unhandled case where we need to cleanup stale rels between SGs and instances.